tests/xharness/Jenkins.cs

```diff
@@ -676,10 +676,6 @@ void DisableKnownFailingDeviceTests ()
 		{
 			// https://github.com/xamarin/maccore/issues/1008
 			ForceExtensionBuildOnly = true;
-
```

Comment:
mini is still mentioned in this file on L318: https://github.com/xamarin/xamarin-macios/blob/5108f5659b4c680f825d94df0eb4f25e75d246e4/tests/xharness/Jenkins.cs#L318-L328

Reply:
@akoeplinger nice catch, I'll remove that